packages/cli/src/utils/files.js

```diff
@@ -98,41 +98,41 @@ const copyDir = (src, dst, options) => {
     onSkip: () => {},
   });
 
-  return ensureDir(dst)
-    .then(() => fse.readdir(src))
-    .then((files) => {
-      const promises = files.map((file) => {
-        const srcItem = path.resolve(src, file);
-        const dstItem = path.resolve(dst, file);
-        const stat = fse.statSync(srcItem);
-        const dstExists = fileExistsSync(dstItem);
-
-        if (!options.filter(srcItem)) {
-          return Promise.resolve();
-        }
-
-        if (dstExists && options.clobber) {
-          fse.removeSync(dstItem);
-        } else if (dstExists) {
-          if (!stat.isDirectory()) {
-            options.onSkip(dstItem);
-            return Promise.resolve();
-          }
-        }
-
-        if (stat.isDirectory()) {
-          return ensureDir(dstItem).then(() =>
-            copyDir(srcItem, dstItem, options)
-          );
-        } else {
-          return copyFile(srcItem, dstItem, stat.mode).then(() => {
-            options.onCopy(dstItem);
-          });
-        }
+  await ensureDir(dst);
+  const files = await fse.readdir(src);
+
+  const promises = files.map(async (file) => {
+    const srcItem = path.resolve(src, file);
+
+    // skip srcItem if filter returns false
+    if (!options.filter(srcItem)) {
+      return Promise.resolve();
```

Comment:
Since we're already in an async function, `Promise.resolve()` is unnecessary.

```suggestion
      return;
```